src/AddressRegistry/StreetName/StreetName.cs

```diff
@@ -61,14 +61,30 @@ public void RetireStreetName()
                 return;
             }
 
+            RejectAddressesBecauseStreetNameWasRetired(
+                StreetNameAddresses.ProposedStreetNameAddresses.Where(address => address.IsBoxNumberAddress));
+            RejectAddressesBecauseStreetNameWasRetired(
+                StreetNameAddresses.ProposedStreetNameAddresses.Where(address => address.IsHouseNumberAddress));
+
+            RetireAddressesBecauseStreetNameWasRetired(
```

Comment:
Can you move this before the housenumbers?

Reply:
Done. That's indeed correct.